src/components/composer/grid_composer/grid_composer.ts

```diff
@@ -109,10 +98,17 @@ export class GridComposer extends Component<Props, SpreadsheetChildEnv> {
       textAlign = style.align || cell.defaultAlign;
     }
 
+    /**
+     * min-size is on the container, not the composer element, because we want to have the same size as the cell by default,
+     * including all the paddings/margins of the composer
+     *
+     * The +-1 are there to include cell borders in the composer sizing/positioning
```

Comment:
I'd say the +1 is correct because we should want the cells borders to be rendered as 1 px wide no ? If the cells borders are more than 1px wide, the issue is there (see [3103398](https://www.odoo.com/web#id=3103398&menu_id=4720&cids=1&action=333&active_id=2328&model=project.task&view_type=form) )

Reply:
First thing first, we should not confuse a cell with the selection, they're not the same. the issue you mention has nothing to do with the selection, and they are not cell borders, they are grid lines.

Now, the drawing of the **selection** border inside the canvas is larger than a grid line, and seems to be on purpose. I think the logic was to have the same width between the trait of the canvas and the composer borderwidth - from what I see in the code, That would be to have a "seamless" transition. For example, GSheet does that. 
As I pointed in my previous message, this is not the case atm because we use static +1 in the computaions and the borderwidth is instanciated once and does not change when the dpr changes.

I feel like we should have the same behaviour as gsheet, so try to have the same trait width for the canvas and the border of the grid composer. **This is clearly an improvement from what we are doing right now** so I'd suggest that we do it right away while we are changeing lots of things in the composer but of course it can be done in another task. Think we should to it here, though

Reply:
https://www.odoo.com/web#menu_id=4720&cids=1&action=333&active_id=2328&model=project.task&view_type=form&id=3140612